workflow.txt

```diff
@@ -1,10 +1,9 @@
 git clone
 git checkout -b my-solution
+git add .
 git commit -m ""
+git push origin my-solution
 git fetch --all
-git add .
-git pull upstream master
-git merge my-solution
-fork on github
 git checkout master
+git merge my-solution
 git push origin my-solution
```

Comment:
close, though for a typical hw submission it would look like:

```
fork on github
git clone
git checkout -b my-solution
git add .
git commit -m ""
git push origin my-solution
```